### PR TITLE
chore: remove tokio-futures facade crate

### PR DESCRIPTION
**Pull request by carllerche**
This switches from using the tokio-futures facade to referencing
futures-* crates directly.

### CODE REVIEW

File: tokio-fs/src/write.rs [hunk 1]
```diff
@@ -1,11 +1,13 @@
 use crate::{file, File};
+
+use tokio_io::AsyncWrite;
```
Comment (jonhoo) [inline tokio-fs/src/write.rs]: Why the newline separation here (and in all the other files)?

Comment (carllerche) [reply tokio-fs/src/write.rs]: I believe fmt alphabetizes on a per group of imports. I think it is most helpful to first see imports from the current crate, then imports from other tokio crates, then "ecosystem" imports.

Comment (carllerche) [reply tokio-fs/src/write.rs]: speaking of which, I didn't fmt.

Comment (jonhoo) [reply tokio-fs/src/write.rs]: Ah, so basically you want https://github.com/rust-lang/rustfmt/issues/2979 :p

Comment (carllerche) [reply tokio-fs/src/write.rs]: yep

**Pull request merged**